translib/acl_app.go

```diff
@@ -168,68 +169,8 @@ func (app *AclApp) translateGet(dbs [db.MaxDB]*db.DB) error {
 }
 
 func (app *AclApp) translateAction(dbs [db.MaxDB]*db.DB) error {
-    err := errors.New("Not supported")
-    return err
-}
-
-func (app *AclApp) translateSubscribe(dbs [db.MaxDB]*db.DB, path string) (*notificationOpts, *notificationInfo, error) {
-	pathInfo := NewPathInfo(path)
-	notifInfo := notificationInfo{dbno: db.ConfigDB}
-	notSupported := tlerr.NotSupportedError{
-		Format: "Subscribe not supported", Path: path}
-
-	if isSubtreeRequest(pathInfo.Template, "/openconfig-acl:acl/acl-sets") {
-		// Subscribing to top level ACL record is not supported. It requires listening
-		// to 2 tables (ACL and ACL_RULE); TransLib does not support it yet
-		if pathInfo.HasSuffix("/acl-sets") ||
-			pathInfo.HasSuffix("/acl-set") ||
-			pathInfo.HasSuffix("/acl-set{}{}") {
-			log.Errorf("Subscribe not supported for top level ACL %s", pathInfo.Template)
-			return nil, nil, notSupported
-		}
-
-		t, err := getAclTypeOCEnumFromName(pathInfo.Var("type"))
-		if err != nil {
-			return nil, nil, err
-		}
-
-		aclkey := getAclKeyStrFromOCKey(pathInfo.Var("name"), t)
-
-		if strings.Contains(pathInfo.Template, "/acl-entry{}") {
-			// Subscribe for one rule
-			rulekey := "RULE_" + pathInfo.Var("sequence-id")
-			notifInfo.table = db.TableSpec{Name: RULE_TABLE}
-			notifInfo.key = asKey(aclkey, rulekey)
-			notifInfo.needCache = !pathInfo.HasSuffix("/acl-entry{}")
-
-		} else if pathInfo.HasSuffix("/acl-entries") || pathInfo.HasSuffix("/acl-entry") {
-			// Subscribe for all rules of an ACL
-			notifInfo.table = db.TableSpec{Name: RULE_TABLE}
-			notifInfo.key = asKey(aclkey, "*")
-
-		} else {
-			// Subscibe for ACL fields only
-			notifInfo.table = db.TableSpec{Name: ACL_TABLE}
-			notifInfo.key = asKey(aclkey)
-			notifInfo.needCache = true
-		}
-
-	} else if isSubtreeRequest(pathInfo.Template, "/openconfig-acl:acl/interfaces") {
-		// Right now interface binding config is maintained within ACL
-		// table itself. Multiple ACLs can be bound to one intf; one
-		// inname can occur in multiple ACL entries. So we cannot map
-		// interface binding xpaths to specific ACL table entry keys.
-		// For now subscribe for full ACL table!!
-		notifInfo.table = db.TableSpec{Name: ACL_TABLE}
-		notifInfo.key = asKey("*")
-		notifInfo.needCache = true
-
-	} else {
-		log.Errorf("Unknown path %s", pathInfo.Template)
-		return nil, nil, notSupported
-	}
-
-	return nil, &notifInfo, nil
+	err := errors.New("Not supported")
```

Comment:
This is an existing code; but showing up here because gofmt converted indentation spaces into tabs. No idea why the original author chose to code it that way. I am not concentrating on such cleanups in this PR